core/Korean/ko-osmanage-ConfiguringSystemSettings.md

```diff
@@ -0,0 +1,912 @@
+<!--
+SPDX-FileCopyrightText: 2023,2024 Oracle and/or its affiliates.
```

Comment:
this should be your copyright, not oracle's imho.

Reply:
I added SPDX-FileCopyrightText in a docs.